Lib/test/test_support.py

```diff
@@ -746,7 +746,18 @@ def test_get_signal_name(self):
 
     def test_linked_to_musl(self):
         linked = support.linked_to_musl()
-        self.assertIsInstance(linked, bool)
+        self.assertIsNotNone(linked)
+        if support.is_wasi:
+            self.assertTrue(linked)
+        # The value is cached, so make sure it returns the same value again.
+        self.assertEqual(linked, support.linked_to_musl())
+        # The unlike libc, the musl version is a triple.
+        if linked:
+            self.assertIsInstance(linked, tuple)
+            self.assertEqual(3, len(linked))
```

Comment:
```suggestion
            self.assertEqual(len(linked), 3)
```

Reply:
Expected should come first because it makes the assertEqual diffs clearer in the general case.  In this case it wouldn't matter, but I prefer to be consistent.

Reply:
Ah, most of the tests I wrote and read were actually swapped. I think it's half-half so I don't mind.